typescript/sdk/src/ism/EvmIsmReader.ts

```diff
@@ -208,9 +216,14 @@ export class EvmIsmReader extends HyperlaneReader implements IsmReader {
       async (module) => this.deriveIsmConfig(module),
     );
 
+    // If it's a zkSync chain, it must be a StorageAggregationIsm
+    const ismType = this.isZkSyncChain
+      ? IsmType.STORAGE_AGGREGATION
```

Comment:
if we have storage aggregations now, do we still need to have the weird setup for avoiding aggregation hooks / ISMs?

Reply:
I believe we have aggregation isms but not aggregation hooks, so at least that can be updated. But I would prefer to do that in a separate PR at the time we actually enact those changes onchain, otherwise we'll get the checker complaining